.github/workflows/ci.yml

```
@@ -705,6 +705,48 @@ jobs:
         run: docker logs earthly-buildkitd
         if: ${{ failure() }}
 
+  load-images-test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        type:
+          - with-docker
+          - with-docker-registry
+        test_image:
+          - alpine:3.15
+          - nginx:1.22.0
+          - node:gallium
+          - aergus/latex
+    env:
+      FORCE_COLOR: 1
+      EARTHLY_TOKEN: "${{ secrets.EARTHLY_TOKEN }}"
+      EARTHLY_INSTALL_ID: "earthly-githubactions"
+    steps:
+      - uses: earthly/actions-setup@v1.0.1
+      - uses: actions/checkout@v3
+      - name: Docker mirror login (Earthly Only)
+        run: docker login registry-1.docker.io.mirror.corp.earthly.dev --username "${{ secrets.DOCKERHUB_MIRROR_USERNAME }}" --password "${{ secrets.DOCKERHUB_MIRROR_PASSWORD }}"
+        if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository
+      - name: Configure Earthly to use mirror (Earthly Only)
+        run: |-
+          earthly config global.buildkit_additional_config "'[registry.\"docker.io\"]
+
+          mirrors = [\"registry-1.docker.io.mirror.corp.earthly.dev\"]'"
+        if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository
+      - name: Build latest earthly using released earthly
+        run: earthly --use-inline-cache +for-linux
+      - name: Set EARTHLY_VERSION_FLAG_OVERRIDES env
+        run: |-
+            set -euo pipefail
+            EARTHLY_VERSION_FLAG_OVERRIDES="$(tr -d '\n' < .earthly_version_flag_overrides)"
+            echo "EARTHLY_VERSION_FLAG_OVERRIDES=$EARTHLY_VERSION_FLAG_OVERRIDES" >> "$GITHUB_ENV"
+      - name: Run load tests
+        run: ./build/linux/amd64/earthly -P ./tests/${{ matrix.type }}/+all --TEST_IMAGE=${{ matrix.test_image }}M --earthly=$(realpath ./build/linux/amd64/earthly)
```

looks like a `M` suffix slipped in.

```suggestion
        run: ./build/linux/amd64/earthly -P ./tests/${{ matrix.type }}/+all --TEST_IMAGE=${{ matrix.test_image }} --earthly=$(realpath ./build/linux/amd64/earthly)
```